frontend/src/App.tsx

```diff
@@ -39,8 +39,8 @@ const App = () => {
               <Route path="*" element={<Page404 />} />
```

Comment:
if we are looking at this, one thing that annoys me is that 404 is rendered if it loads first, which is irritating. can we improve this routing to not have that happen?

Reply:
https://stackoverflow.com/questions/54625911/404-page-in-react-renders-in-every-page I think this might help?